src/app/app.module.ts

```diff
@@ -126,6 +126,7 @@ import { FilterEcoNewsPipe } from './pipe/filter-ecoNews-pipe/filter-eco-news.pi
 import {InfiniteScrollModule} from 'ngx-infinite-scroll';
```

Comment:
![image](https://user-images.githubusercontent.com/54113846/77180877-4070cf00-6ad3-11ea-8e94-098ffda73bb3.png)

Reply:
please use date pipe('longDate')

Reply:
fixed, used standard format according to Figma